systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -106,11 +111,36 @@
 @Tag(CONNECT)
 @Tag(CONNECT_COMPONENTS)
 @SuppressWarnings({"checkstyle:ClassDataAbstractionCoupling"})
+@SuiteDoc(
+    description = @Desc("Verifies the deployment, manual rolling update, and undeployment of Kafka Connect components."),
+    beforeTestSteps = {
+        @Step(value = "Deploy uber operator across all namespaces, with custom configuration", expected = "Uber operator is deployed"),
```

Comment:
uber operator is a bit misleading in this case IMHO
I would just use Cluster Operator or something

uber operator, for me, is operator deploying other operators, which deploy another components.

Reply:
Removed all before test steps here.